tests/secrets/secrets_vault_test.go

```diff
@@ -0,0 +1,106 @@
+package tests
```

Comment:
Let's move this to https://github.com/portworx/torpedo/blob/master/tests/basic/basic_test.go. We currently have a problem in torpedo where a new testsuite will increase the size of image and the build time. Until that's fixed, we will try to avoid adding new testsuite folders.